### DISCUSSION TITLE
remove searx results with minus sign

### DISCUSSION THREAD

**Discussion by DiamondDemon669** [searx/searx#3229] [Development]
I want to remove results with the '-' sign like you can do on google. Can you point me in the direction I would need to go to implement this?

I also want to implement string grouping (make sure "words needed together" are searched together) and site: filtering but the minus sign is most important

**Comment by unixfox** [comment]
For `site:` like feature there is an issue opened here: https://github.com/searx/searx/issues/2952

By the way searx is in maintenance only project, you might want to check out SearXNG which has an active development and plenty of active devs: https://github.com/searxng/searxng

Realtime chat about SearXNG available here: https://github.com/searxng/searxng#contact

**Reply by kvch** [reply]
For the record, searx is not maintenance only, it accepts new features if they are privacy respecting.

**Comment by kvch** [comment]
The change was merged to searx: https://github.com/searx/searx/commit/a825690804e2f144d57715a41193b036dd206d0f
